degiroapi/__init__.py

```diff
@@ -36,6 +36,12 @@ class DeGiro:
     client_info = any
     confirmation_id = any
 
+    def __init__(self, username=None, password=None, totp=None):
+        if not username: # Proceed without login
+            pass
+        else: # Login prompt
+            DeGiro.login_prompt(self, username=username, password=password, totp=totp)
```

Comment:
```suggestion
            self.login_prompt(username=username, password=password, totp=totp)
```
This should work the same, but it's the canonical way to call class methods.